.github/workflows/aws-docker-deploy.yml

```diff
@@ -1,5 +1,7 @@
 name: AWS Docker Deploy
```

Comment:
all the worklfows are "doubled" here, I think I could simply remove the old ones as they are not "used" by anyone, contrarily to the actions below.. (aka this is specific for this repo, and pretty simple renaming?)

Reply:
Makes sense since these are workflows used only within this repo

Reply:
Is this workflow `.github/workflows/aws-docker-deploy.yml` used anywhere? I think it should be an action if it is